includes/class-edd-fees.php

```diff
@@ -97,7 +97,10 @@ public function add_fee( $args = array() ) {
 		$args['amount'] = edd_sanitize_amount( $args['amount'] );
 
 		// Set the fee
-		$fees[ $key ] = $args;
+		$fees[ $key ] = apply_filters( 'edd_fees_add_fees', $args, $this );
```

Comment:
Yes, I probably copy/pasted the filter names and forgot to adjust it. Would you like me to change it?